os/fs/driver/mtd/smart.c

```diff
@@ -5318,9 +5320,9 @@ static int smart_journal_bwrite(FAR struct smart_struct_s *dev, uint16_t physsec
  * Name: smart_journal_bytewrite
  *
  * Description:
- *   Byte Write function instead of smart_bytewrite, return is size of written bytes.
- *   It is called when smartfs release target sector only, it means no data.
- *   This will not write data on physical sector.
+ *   Byte Write function instead of smart_bytewrite, returns size of written
+ *   bytes. It is called when smartfs releases target sector only. This
+ *   function will not write data bytes to the physical sector.
```

Comment:
This   function will not write data bytes to the physical sector.
=> This function will write only 1 byte to the physical sector
It will be better, how do you think?

Reply:
Modified and made clear.